tests/test_create.py

```diff
@@ -310,3 +325,39 @@ def test_creator_info(fs, simple_mhl_history):
     assert "franz@example.com" in result.output
     assert "123-4567" in result.output
     assert "Data Manager" in result.output
+
+
+def test_create_mulitple_hashformats(fs, simple_mhl_history):
+    runner = CliRunner()
+    result = runner.invoke(ascmhl.commands.create, ["/root", "-v", "-n", "-h", "md5", "-h", "sha1"])
+    assert result.exit_code == 0
+
+    assert "A/A1.txt  md5: fe6975a937016c20b43b17540e6c6246" in result.output
+    assert "A/A1.txt  sha1: 4a5b95edbea7de5ed2367432645df88cd4f1d1b6" in result.output
+
+
+def test_create_mulitple_hashformats_no_dash_n(fs, simple_mhl_history):
+    runner = CliRunner()
+    result = runner.invoke(ascmhl.commands.create, ["/root", "-v", "-h", "md5", "-h", "sha1"])
+    assert result.exit_code == 0
+
+    assert "A/A1.txt  md5: fe6975a937016c20b43b17540e6c6246" in result.output
+    assert "A/A1.txt  sha1: 4a5b95edbea7de5ed2367432645df88cd4f1d1b6" in result.output
+
+
+@freeze_time("2020-01-16 09:15:00")
+def test_create_mulitple_hashformats_double_hashformat(fs, simple_mhl_history):
```

Comment:
This test is still failing for me locally.  Is this expected behavior?

Reply:
I think `pytest .` actually runs the test successfully (e.g. added log output to that test that was printed). But I also experienced the test failing from within PyCharm – but also with a wrong line number of the assert. So I would trust the pytest command and the test on github for now that it runs successfully.

Reply:
You are correct.  running pytest . completes successfully.  Appears to be a problem with Pycharm